include/poketch_data.h

```diff
@@ -49,245 +54,149 @@ enum PoketchScreenColor {
- *                          the month will update to this value and clear all marks on the calendar.
- * @param day:              The day of the date to clear.
+ * Passing this function a new month will change the month on the calendar and clear every day.
  */
 void PoketchData_ClearCalendarMark(PoketchData *poketchData, u32 month, u32 day);
 
-/**
- * Checks whether the given date is currently marked on the calendar.
- *
- * @param poketchData:      The Poketch data to check.
- * @param month:            The month of the date to check.
- * @param day:              The day of the date to check.
- * 
- * @param return TRUE if the date is marked, FALSE if not. If the given month is not the currently stored month, returns FALSE.
- */
 BOOL PoketchData_CalendarMarked(const PoketchData *poketchData, u32 month, u32 day);
 
 /**
- * Sets the location of a map marker.
- *
- * @param poketchData:      The Poketch data to update.
- * @param index:            The Map Marker to set. This function asserts that the index is less than POKETCH_MAPMARKER_COUNT.
- * @param x:                The x coordinate for the map marker.
- * @param y:                The y coordinate for the map marker.
+ * Sets the location of a map marker. Asserts that index is less than POKETCH_MAPMARKER_COUNT.
  */
 void PoketchData_SetMapMarker(PoketchData *poketchData, int index, u8 x, u8 y);
 
 /**
- * Gets the location of a map marker.
- *
- * @param poketchData:      The Poketch data to update.
- * @param index:            The Map Marker to check. This function asserts that the index is less than POKETCH_MAPMARKER_COUNT.
- * @param x:                The int where the x coordinate will be stored.
- * @param y:                The int where the y coordinate will be stored.
+ * Gets the location of a map marker. Asserts that index is less than POKETCH_MAPMARKER_COUNT.
  */
 void PoketchData_MapMarkerPos(const PoketchData *poketchData, int index, u8 *x, u8 *y);
 
-BOOL sub_02056A10(const PoketchData *poketchData);
-void sub_02056A18(const PoketchData *poketchData, u8 *param1);
-void sub_02056A2C(PoketchData *poketchData, const u8 *param1);
-void sub_02056A48(PoketchData *poketchData, const BoxPokemon *param1);
-int sub_02056AAC(const PoketchData *poketchData);
-void sub_02056AC8(const PoketchData *poketchData, int param1, int *param2, int *param3);
-u32 sub_02056AFC(const PoketchData *poketchData, int param1);
+/**
+ * Checks whether or not the Dot Art data has been modified at any point. 
+ * This will return FALSE until PoketchData_SetDotArtData is called, at which point it will always return TRUE.
+ */
+BOOL PoketchData_DotArtModified(const PoketchData *poketchData);
+
+void PoketchData_DotArtData(const PoketchData *poketchData, u8 *dst);
+
+void PoketchData_SetDotArtData(PoketchData *poketchData, const u8 *src);
+
+/**
+ * Adds a new Pokemon to the end of the Pokemon History list.
+ */
+void PoketchData_PokemonHistoryAddEntry(PoketchData *poketchData, const BoxPokemon *boxPokemon);
```

Comment:
`PoketchData_PokemonHistoryEnqueue`? That might not be clear to folks without a formal comp-sci background.